retico_rasanlu/rasanlu.py

```diff
@@ -126,19 +168,37 @@ def process_revoke(self, revoked_iu):
         
         result =  None
         if self.incremental:
-            for word in reversed(revoked_iu.get_text().split()):
-                text_iu = (word, "revoke") 
-                # print('nlu revoke({})'.format(word))
-                result = self.interpreter.parse_incremental(text_iu)
-            if result is not None:
-                result = self.process_result(result, revoked_iu)
+
+            tokens = tokens=self.preproccessor(revoked_iu.get_text()).split()
```

Comment:
was this duplication of "tokens=" intentional?

Reply:
Definitely a mistake, I'll fix that.